data/os/RedHat/8.yaml

```diff
@@ -0,0 +1,15 @@
+---
+oxidized::ruby_dependencies:
+  - ruby
+oxidized::install_dependencies:
```

Comment:
Is the override for this still needed if you pull libssh2 from EPEL?

Reply:
Some are required for git gem build and RHLE8 uses a hardened compiler which is shipped as part of redhat-rpm-config